bindings/nodejs/lib/types/models/api/info/node-info.ts

```diff
@@ -20,17 +20,17 @@ export interface INodeInfo {
     /**
      * The status of the node.
      */
-    status: INodeInfoStatus;
+    status: NodeInfoStatus;
```

Comment:
Since you've removed the Node prefix in InfoResponse, I'd personally also do it for the contained one, better match Rust as well

Reply:
done (also for python)